src/packs/hipaa-security.ts

```diff
@@ -310,9 +310,9 @@ export class HIPAASecurityChecks extends NagPack {
       node: node,
     });
     this.applyRule({
-      info: 'The CodeBuild project which utilizes either a GitHub or BitBucket source repository does not utilize OAUTH - (Control IDs: 164.308(a)(3)(i), 164.308(a)(4)(ii)(A), 164.308(a)(4)(ii)(C), 164.312(a)(1)).',
```

Comment:
For consistency can we also update the description for these two packs (and their RULES.md entries)?

1. https://github.com/quixoticmonk/cdk-nag/blob/rules_info_wording/src/packs/nist-800-53-r4.ts#L263
2. https://github.com/quixoticmonk/cdk-nag/blob/rules_info_wording/src/packs/pci-dss-321.ts#L279